x/market/hooks/hooks.go

```
@@ -41,17 +41,29 @@ func (h *hooks) OnEscrowAccountClosed(ctx sdk.Context, obj etypes.Account) {
 
-	gstate := dtypes.GroupClosed
-	if obj.State.State == etypes.StateOverdrawn {
-		gstate = dtypes.GroupInsufficientFunds
+	var gstate dtypes.Group_State
+
+	switch obj.State.State {
+	case etypes.StateOverdrawn:
+		gstate = dtypes.GroupPaused
+	default:
+		gstate = dtypes.GroupClosed
+		h.dkeeper.CloseDeployment(ctx, deployment)
```

golangci-lint (2.5.0)

[error] Error return value of `h.dkeeper.CloseDeployment` is not checked

(errcheck)